.github/ISSUE_TEMPLATE/08-unstructured.yml

```diff
@@ -0,0 +1,12 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: "Internal: unstructured"
```

Comment:
nit: I think "blank" is slightly more obvious, but defer entirely to you on this

Reply:
Oh, no, no strong feels! Updated.